components/ResultsPageComponents/DistanceTravelledMode/DistanceTravelledMode.jsx

```diff
@@ -12,124 +12,136 @@ export default function DistanceTravelledMode({ data }) {
       minWidth="350px"
       maxWidth="1100px"
       direction="column"
+      alignSelf={["center", "start"]}
+      align={["center", "flex-start"]}
+      flex={[1, 2]}
       gap={["10px", "20px"]}
-      px={["10px", "10px"]}
+      px={["5px", "50px"]}
+      py={["25px", "50px"]}
+      justify="center"
     >
-      <Flex direction="column">
+      <Flex direction="column" width="100%" py="15px">
         <Text fontSize="27.65px" fontWeight={600}>
           Distance travelled by mode of transport
         </Text>
         <Text fontSize="20px">
-          This graph(butterfly plot) help us compare the travel modes(individual
-          Vs.active/active/public/share)preferred by staff within various
-          commute distances.
+          The following graph helps us compare the travel modes (individual
+          vs.active/public/shared) preferred by staff within various commute
+          distances.
         </Text>
       </Flex>
 
       {/* CONTAINER FOR INDIVIDUAL METHOD AND ACTIVE/PUBLIC/SHARED/METHOD   */}
-      <Flex direction={["column", "row"]} width="100%" my="15px">
+      {/* <Flex direction="column">
+      </Flex> */}
+      <Flex direction="column" gap="10px" mb="10px">
         <Flex
-          direction="column"
-          width={["100%", "50%"]}
-          align={["left", "center"]}
-          justifyContent="space-between"
+          gap="30px"
+          direction={["column", "row"]}
+          justify={["center", "space-between"]}
+          align="center"
         >
-          {/* row for the sub title */}
-          <Flex direction="row" align="center" my="5px">
-            <Flex>
-              <Box
-                mr="5px"
-                borderRadius="2px solid #E6EEF3"
-                width="17px"
-                height="17px"
-                background="#D69E2E"
-              ></Box>
-            </Flex>
-            <Flex>
-              <Text fontWeight={600} fontSize="23px">
-                Individual Method
-              </Text>
-            </Flex>
+          <Flex direction="column">
+            <Text fontWeight={600} color="#022640" fontSize="22px">
+              Active/Public/Shared Methods:
+            </Text>
+            <Text color="#03385F">
+              Lower carbon emission per kilometre travel per person
```

Comment:
smaller grammar fix:
```suggestion
              Lower carbon emission per kilometres travelled per person
```